chiselFrontend/src/main/scala/chisel3/core/Aggregate.scala

```diff
@@ -237,8 +237,8 @@ sealed class Vec[T <: Data] private (gen: => T, val length: Int)
   }
 
   private[chisel3] def toType: String = s"${sample_element.toType}[$length]"
-  private[chisel3] lazy val flatten: IndexedSeq[Bits] =
-    (0 until length).flatMap(i => this.apply(i).flatten)
+  override def getElements: Seq[Data] =
+    (0 until length).map(apply(_))
```

Comment:
I don't think there's a reason to re-define the ScalaDoc in override functions in subclasses.